README.md

```diff
@@ -23,12 +23,12 @@ Good documentation regarding 'systemd-boot' can be found here:
 The helper scripts in the main folder do the following:
 
 * Download and prepare third party software dependencies, e.g. the precompiled UEFI boot loader images, OVMF images, Syslinux, sample kernel/initramfs files, etc. 
-* Genrate sample 'El Torito' boot image as described in UEFI sepcification 2.7, sections 13.3.1.x and 13.3.2.x.
+* Genrate sample 'El Torito' boot image as described in UEFI specification 2.7, sections 13.3.1.x and 13.3.2.x.
 * Generate sample ISO image with UEFI boot support.
 * Run [QEMU](https://www.qemu.org) with UEFI enabled configuration and attached sample ISO image.
 
 Note that you need QEMU on your system in order to test the sample ISO images. You can install it like this (Ubuntu):
 
-* ``sudo apt install qemu``
+* ``sudo apt-get update && sudo apt install qemu``
```

Comment:
Why do you introduce ``apt-get`` here? You can use ``apt`` in the same way.

Reply:
Isn't it good practice to do the update before the install? Sometimes, the install may not even be found. (depending on whether apt sources have been changed or not)

Reply:
It is a good practice, but you suggest using ``apt-get`` instead of ``apt``. Mixing two different styles is definitely not a good practice. Change it to ``apt`` and I'll merge the PR. :)

Reply:
Also, you might want to check and update this README as well:

https://github.com/ivandavidov/systemd-boot/tree/master/project

Reply:
Good catch on the other